README.md

```diff
@@ -75,9 +75,9 @@ Select 1:
 Identify the three places CSS can go, and rank them in terms of specificity:
 
 ```text
-1.
-2.
-3.
+1. External Stylesheet
+2. Internal Stylesheet
+3. Inline Style
```

Comment:
Close.  This is the appropriate order, if you reverse it.  Inline style has the most specificity, but is least desirable.